sdks/java/core/src/main/java/org/apache/beam/sdk/values/PCollectionViews.java

```diff
@@ -1403,4 +1427,55 @@ public int size() {
       };
     }
   }
+
+  public static <InputT, ViewT> PCollectionView<ViewT> findPCollectionView(
```

Comment:
Can we choose not to expose this as it looks like it is only used within PipelineTest.java

If there is a future need we can move it again (in hopefully a much smaller change) so its easier to reason about visibility.

Reply:
Yea. This is actually a simplified version of something that exists in `DataflowRunner.java`. I did not want to risk the stability of this change by also altering it there, but they should be merged.